doc/REQUIREMENTS.MD

```diff
@@ -1,3 +1,18 @@
+- Create order API (called from `CREATEORDER_ENDPOINT_URL`) should be implemented as a GET service which accepts `encryptedCartId` query parameter with Blowfish encrypted cart id string, for instance:
+
+  ```
+  http://localhost:9000/API/order/createOrder?encryptedCartId=eDIeqi7seRTjdYRqm%2BOxy5Qcbc38dXaEgftHjQcr6M7HO125Tg%2BnUg%3D%3D
+  ```
```

Comment:
finally we figured it out it is already base64Encoded, and should be than Url encoded :)